setup.py

```diff
@@ -57,6 +52,7 @@ def find_qiskit_aer_packages():
         "Topic :: Scientific/Engineering",
     ],
     install_requires=requirements,
+    setup_requires=['scikit-build', 'cmake', 'ninja', 'Cython'],
```

Comment:
We don't need `ninja`, not even on Windows. scikit-build tries `ninja` as the first option always (I think this is going to change soon) , but if `ninja` is not found, it jumps to the next toolchain.

Reply:
I just copied what was in the pyproject.toml file I can change this list to be whatever set of tools you think makes sense.

Reply:
Ok, just removes `ninja` and I'll be merging
Thanks!

Reply:
Fixed in: https://github.com/Qiskit/qiskit-aer/pull/401/commits/f344eedd46038414a38da62d1b33b1de080c88a3